src/main/helm/helm-repo-manager.ts

```diff
@@ -146,7 +146,7 @@ export class HelmRepoManager extends Singleton {
     return stdout;
   }
 
-  public static async addСustomRepo(repoAttributes : HelmRepo) {
+  public static async addCustomRepo(repoAttributes : HelmRepo) {
```

Comment:
I think this was Cyrillic although I didn't check. n.b. not all instances of `addCustomRepo` in the files touched by this commit used the non-ascii `C`.